apps/pie-docs/src/foundations/spacing/overview.md

```diff
@@ -10,47 +10,47 @@ There are several types of spacing we can use to place elements within layouts a
+caption: "The example above shows a 24px left and right padding, which is part of our Button component."
 } %}
 
 ### Margins
 
-Margins are the distance between components, which can be applied vertically and horizontally. They define the relationship between components, layout and viewport size.
+Margins are the distances between components and can be applied vertically and horizontally. They define the relationship between components, layout and viewport size.
 
 {% contentPageImage {
 src:"../../../assets/img/foundations/spacing/margin-01.svg",
 width: "224px",
-caption: "The example above shows a 16px margin between a Secondary and Primary button."
+caption: "The example above shows a 16px margin between two buttons."
 } %}
 
 ### Dimensions
 
-Dimensions describe the overall width and height of a component or design element. While vertical dimensions should always stick to our spacing scale, horizontal dimensions won’t be specified because they are responsive to the viewport’s width.
+Dimensions describe the overall width and height of a component or design element. While vertical dimensions should always stick to our spacing scale, horizontal dimensions are not specified because they are responsive to the viewport’s width.
 
 {% contentPageImage {
 src:"../../../assets/img/foundations/spacing/dimensions-01.svg",
 width: "360px",
-caption: "The example above shows how a Primary Button’s width spans across 328px to fit the width of the X-Small Layout."
+caption: "The example above shows how a primary button’s width spans across 328px to fit the width of the x-small layout."
 } %}
 
 ---
 
 ## Element first approach
 
-Our main approach for designing components can be categorised as ‘element first’. This means that the sizing of components and design elements takes priority when matched to our predetermined spatial system. We follow this approach to keep a consistent rythm within the overall composition and layout of our designs.
+Our main approach for designing components can be categorised as ‘element first’. This means that the sizing of components and design elements takes priority when matched to our predetermined spatial system. We follow this approach to keep a consistent rhythm within the overall composition and layout of our designs.
 
-For instance, an element might be set to always have a height of 48px, regardless of the content that can be found inside it. This means that if the component has different variations (like different typographic sizes) it will always preserve a 48px height, and it’s the internal padding that’s going to change to accommodate the difference in height.
+For instance, an element might be set to always have a height of 48px, regardless of the content found inside it. This means that if the component has different variations (like different typographic sizes) it will always preserve a 48px height, and it’s the internal padding that’s going to change to accommodate the difference in height.
```

Comment:
```suggestion
For instance, an element might be set to always have a height of 48px, regardless of the content found inside it. This means that if the component has different variations (like different typographic sizes) it will always preserve a 48px height, and the internal padding will change to accommodate the difference in height.
```

Reply:
👍